pom.xml

```diff
@@ -14,7 +14,7 @@
     <parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
-        <version>2.6.1</version>
+        <version>2.6.8</version>
```

Comment:
I like to keep up-to-date versions, but if it is not strictly required by the feature branch, I would prefer to have such changes in a separate PR.

Reply:
This is a bugfix version update, I think we can do these more casually...